packages/experimental/Expander/api-extractor.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@uifabricshared/build-native/api-extractor.json"
```

Comment:
nit shouldn't be needed w/ api-extractor removal

Reply:
Good catch! We can (and should) delete this file now that we don't use it.